automation/src/test/java/org/greenplum/pxf/automation/BaseTestParent.java

```diff
@@ -224,9 +231,14 @@ public void runAfterMethod() throws Exception {
      */
     protected void runTincTest(String tincTest) throws Exception {
         try {
-            tinc.runTest(tincTest);
+            if (TincUtils.useTinc){
+                tinc.runTest(tincTest);
+            } else {
+                regress.runRegressTest(tincTest);
```

Comment:
```suggestion
                regress.runTest(tincTest);
```
let's change the function name for symmetry